satpy/readers/netcdf_utils.py

```diff
@@ -196,11 +210,8 @@ def _get_required_variable_names(listed_variables, variable_name_replacements):
 
     def __del__(self):
         """Delete the file handler."""
-        if self.file_handle is not None:
-            try:
-                self.file_handle.close()
-            except RuntimeError:  # presumably closed already
-                pass
+        if self.manager is not None:
+            self.manager.close()
```

Comment:
Is it worth it to wrap this with a try/except?

Reply:
Maybe?  I figured the `RuntimeError` could not happen because the `CachingFileManager` does not fail when close is called multiple times.  That leaves other exceptions, but I'm not sure what we would do in the `except:`-block, nor what actually happens if an exception is uncaught in `__del__`.

Reply:
Yeah I'm not sure either. Let's see what happens I guess.